tests/support/redis.tcl

```diff
@@ -281,11 +281,7 @@ proc ::redis::redis_read_reply {id fd} {
             ~ -
             * {return [redis_multi_bulk_read $id $fd]}
             % {return [redis_read_map $id $fd]}
-            | {
-                # ignore attributes for now (nowhere to store them)
-                redis_read_map $id $fd
-                continue
-            }
```

Comment:
@sjpotter The non-module test failed due to the change of this piece of code.
I haven't delved int why.

Reply:
yea, figured it out :)  its because the tcl code can't output attribute types with attribute and response together.